### PR TITLE
Dockerization

### PR DESCRIPTION
**Pull request by codegard**
* created Dockerfile for each service
* added separation between integration and unit test

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      replicas: 2
+    ports:
+      - "${SCRAPPER_PORT}:8080"
+    environment:
+      - SPRING_PROFILES_ACTIVE=${ACTIVE_SCRAPPER_PROFILE}
+  db:
+    build: db
+    image: bookfinder/db:latest
+    environment:
+      - POSTGRES_USER=user
```
Comment (smolamarcin) [inline docker-compose.yml:16]: Try to move this sensitive data to the secure place. For this moment it's enough, but keep it in mind.

File: scrapper/.dockerignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Created by .ignore support plugin (hsz.mobi)
```
Comment (smolamarcin) [inline scrapper/.dockerignore:1]: 👍

File: scrapper/pom.xml [hunk 3]
```diff
@@ -3,106 +3,151 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>BookFinder</artifactId>
-        <groupId>pl.codebrothers</groupId>
-        <version>1.0-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.0.5.RELEASE</version>
```
Comment (smolamarcin) [inline scrapper/pom.xml:8]: Maybe try to import Spring boot version from properties?

File: scrapper/src/main/resources/application-prod.properties [hunk 4]
```diff
@@ -1,2 +1,9 @@
-logging.level.org.springframework.web=warn
-logging.level.org.hibernate=warn
\ No newline at end of file
+spring.datasource.url=jdbc:postgresql://db:5432/scrapper
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.username=user
```
Comment (smolamarcin) [inline scrapper/src/main/resources/application-prod.properties:3]: I don't know where you should store this data, but propably not here :D

**Review by smolamarcin — APPROVED** [review]

File: scrapper/pom.xml (hunk 3 above)
Comment (codegard) [reply scrapper/pom.xml:8]: Thank you, it will be improved in the next pull request

**Pull request merged**